src/views/Request.svelte

```diff
@@ -58,7 +62,12 @@ function goToConversation(conversationId) {
       <div class="row">
         <div class="col">
           <h3 class="card-title">{ request.title || ''}</h3>
-          <p>Deliver to <u>{ destination }</u></p>
+          <p>
+            Deliver to <u>{ destination }</u>
+            {#if origin }
+              <br />from <u>{ origin }</u>
```

Comment:
I'd prefer a `div` instead of the `br` here as well as the span with appropriate class for the underline.